01_Data/src/interfaces/repositories.ts

```diff
@@ -133,4 +134,5 @@ export interface IChargingProfileRepository extends CrudRepository<ChargingProfi
   createOrUpdateChargingProfile(chargingProfile: ChargingProfileType, stationId: string, evseId?: number, chargingLimitSource?: ChargingLimitSourceEnumType, isActive?: boolean): Promise<ChargingProfile>;
   createChargingNeeds(chargingNeeds: NotifyEVChargingNeedsRequest, stationId: string): Promise<ChargingNeeds>;
   findChargingNeedsByEvseDBIdAndTransactionDBId(evseDBId: number, transactionDataBaseId: number): Promise<ChargingNeeds | undefined>;
+  createCompositeSchedule(compositeSchedule: CompositeScheduleType, stationId: string): Promise<CompositeSchedule>;
```

Comment:
Out of interest, why did we add it to the ChargingProfileRepository interface? The implementation doesn't really use that repository at all. Or are we structuring it towards our modules?

Reply:
> Or are we structuring it towards our modules?

Yes, but not exactly toward the modules. According to the OCPP, CompositeSchedule is calculated based on ChargingProfiles. They belongs to the same topic. So to keep the code clean, I put them inside the same data model folder and use `ChargingProfileRepository` for CompositeSchedule as well.